### ISSUE TITLE
Unique ID issues

### ISSUE THREAD

**Issue by knakaj** [RunestoneInteractive/thinkcspy#247]
**What Course are you in**
thinkcspy

**Describe the bug**
When running pretext build web, a lot of  @xml:id values in different sections are not unique. 

<img width="959" alt="Screen Shot 2022-07-08 at 3 16 18 PM" src="https://user-images.githubusercontent.com/69680783/178056531-07ab14f4-ad30-450c-892c-88cb4dcd84b3.png">

**Comment by bnmnetp** [comment]
Yes, it looks like many of them are id1.  Further it looks like whatever is generating these ids is creating them without knowing about any of the other files.  within a file it looks like we have id1, id2, id3 etc but we may have multiple id1s in several files.

I have an idea...

In the fixIds.py script we can 

1. Create a counter variable at the top of the inner loop.  (so we are essentially counting files)
2. then we can look for xml:id="id1" , xml:id="id2", etc, etc, in each file and make them look like:   f"xml:id="{counter}_id1"

that should make them all unique.    We should not worry about this too much as I don't think any of these are actually referenced.  But it will clear out all of the error messages about duplicate ids
